editor/inserter/index.js

```diff
@@ -27,13 +27,7 @@ class Inserter extends wp.element.Component {
 		};
 	}
 
-	toggle( event ) {
-		// When opening the menu, track reference to the current active element
```

Comment:
I think some of the inline documentation could be adapted for use in the new higher-order variant.